contracts/base/GuardManager.sol

```diff
@@ -81,11 +91,11 @@ abstract contract GuardManager is SelfAuthorized, IGuardManager {
     }
 
     /**
-     * @dev Internal method to retrieve the current guard
-     *      We do not have a public method because we're short on bytecode size limit,
-     *      to retrieve the guard address, one can use `getStorageAt` from `StorageAccessible` contract
-     *      with the slot `GUARD_STORAGE_SLOT`
-     * @return guard The address of the guard
+     * @notice Internal method to retrieve the current guard.
+     * @dev We do not have a public method because we're short on bytecode size limit,
+     *      to retrieve the guard address, one can use {getStorageAt} from {StorageAccessible} contract
```

Comment:
this statement is not true now because code size now is:
`Safe 21521 bytes (limit is 24576)`

Reply:
I think it is still true - we are just not as short as before.